cmd/agent/app/jmx.go

```diff
@@ -82,7 +82,7 @@ var (
 )
 
 func init() {
-	jmxCmd.PersistentFlags().StringVarP(&jmxLogLevel, "log-level", "l", "debug", "set the log level")
+	jmxCmd.PersistentFlags().StringVarP(&jmxLogLevel, "log-level", "l", "debug", "set the log level for jmxfetch")
```

Comment:
We pass that parameter to the jmxfetch binary when we call it, so I opted to keep the flag and mention it's meant to be passed to jmxfetch.

Reply:
I must miss something then, where do we pass that value? I though it was just the logging level for that command, that's why I though it would be better to have a behaviour similar to the check command change.

Reply:
My bad, it's not passed to jmxfetch. I will do the same as in the check command.